osu.Game/Overlays/Music/PlaylistOverlay.cs

```diff
@@ -102,7 +102,7 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
-            beatmapSubscription = realm.RegisterForNotifications(r => r.All<BeatmapSetInfo>().Where(s => !s.DeletePending), beatmapsChanged);
+            beatmapSubscription = realm.RegisterForNotifications(r => r.All<BeatmapSetInfo>().Where(s => !s.DeletePending && !s.Protected), beatmapsChanged);
```

Comment:
It was originally an intentional design choice to show protected tracks here so users could play them, but I'm okay with changing this for now (at least until the protected beatmaps have playable `.osu` files attached.